lib/ports.json

```diff
@@ -25391,6 +25391,22 @@
       "USPCA"
     ]
   },
+  "USPCV": {
+    "name": "Port Canaveral",
+    "city": "Brevard County",
```

Comment:
not sure about the city, this is a county

Reply:
As far as I understand it's from Antwerp, Belgium :)

Reply:
???
Doesn't make sense the first 2 char of the unloc is US so it's a US port not a Belgium one